package.json

```diff
@@ -45,5 +45,10 @@
 		"matcha": "^0.7.0",
 		"tsd": "^0.17.0",
 		"xo": "^0.44.0"
+	},
+	"xo": {
+		"rules": {
+			"complexity": "off"
```

Comment:
The complexity is now at 29. Should I increase the threshold instead of turning the rule off?